app/importers/activity_insight_importer.rb

```diff
@@ -154,7 +154,7 @@ def call
             pub_record = pi.publication
 
             if pi.persisted?
-              pub_record.update!(pub_attrs(pub)) if pub_record.updated_by_user_at.blank?
+              update_pub_record(pub_record, pub)
             else
```

Comment:
I'd argue that this logic (both what was here previously and the new changes) actually belong in the `Publication` model since we're asking the publication something about its state and then telling it do one thing or another depending on the answer ([Tell Don't Ask](https://www.martinfowler.com/bliki/TellDontAsk.html)). Something like `pub_record.update_from_ai_import(ai_pub)` could maybe contain all of this logic. There's no need to do anything about this now, but it might be a good refactoring for the future, especially since there are probably similar situations in some of the other importers.